modules/guides/examples/javascript-udfs/execute-scoped-function.sh

```diff
@@ -0,0 +1,3 @@
+curl -v http://localhost:8093/query/service \
+  -u Administrator:password \
+  -d 'statement=EXECUTE FUNCTION default:`travel-sample`.`inventory`.GetBusinessDays("03/10/2022", "05/10.2022")'
```

Comment:
Best practice: semicolon at end of N1QL statement

```suggestion
  -d 'statement=EXECUTE FUNCTION default:`travel-sample`.`inventory`.GetBusinessDays("03/10/2022", "05/10.2022");'
```